### PR TITLE
Theming Storybook and adding Viewport

### PR DESCRIPTION
**Pull request by meloneminze**
In this pull request i themed Storybook and added viewport to Storybook to see the components in my default view.

### CODE REVIEW

File: client/.storybook/config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { addDecorator } from '@storybook/react';
+import themeDecorator from './themeDecorator';
+import { addParameters } from '@storybook/react';
```
Comment (marwinburesch) [inline client/.storybook/config.js:3]:
```suggestion
import { addDecorator, addParameters } from '@storybook/react';
import themeDecorator from './themeDecorator';
```
you can consolidate these

File: client/.storybook/daylightTheme.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { create } from '@storybook/theming/create';
+
+export default create({
+  base: 'light',
+
+  textPrimaryColor: '#4D4C4C',
+  textSecondaryColor: '#707070',
+  textThirdColor: '#FFFFFF',
```
Comment (marwinburesch) [inline client/.storybook/daylightTheme.js:8]: following your convention this should be `textTertiaryColor`

File: client/.storybook/daylightTheme.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  backgroundPrimaryColor: '#00A8CC',
+  buttonPrimaryColor: '#45BFD9',
+  warningColor: '#FEA630',
+  lightColor: '#FFBE68',
+  inputColor: '#EFEAEA',
+
+  colorPrimary: 'hotpink',
+  colorSecondary: 'deepskyblue',
```
Comment (marwinburesch) [inline client/.storybook/daylightTheme.js:16]: you other names are a bit more specific than these, is that intentional?

File: client/.storybook/daylightTheme.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+  brandTitle: 'My famly storybook',
+  brandUrl: 'https://github.com/meloneminze/famly',
```
Comment (marwinburesch) [inline client/.storybook/daylightTheme.js:19]: is this required for the `create()` method?

File: client/.storybook/themeDecorator.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
+import GlobalStyles from '../src/components/GlobalStyles';
+import daylight from '../src/themes/daylight';
+
+const ThemeDecorator = (storyFn) => (
+  <ThemeProvider theme={daylight}>
+    <GlobalStyles />
```
Comment (marwinburesch) [inline client/.storybook/themeDecorator.js:8]: it's better to add the `GlobalStyles` component in the `.storybook/config.js` it doesn't really belong in the ThemeDecorator

File: client/.storybook/config.js (hunk 1 above)
Comment (meloneminze) [reply client/.storybook/config.js:3]: Sure!

File: client/.storybook/daylightTheme.js (hunk 3 above)
Comment (meloneminze) [reply client/.storybook/daylightTheme.js:16]: No, these both I should delete.

File: client/.storybook/daylightTheme.js (hunk 2 above)
Comment (meloneminze) [reply client/.storybook/daylightTheme.js:8]: 👍🏼

File: client/.storybook/daylightTheme.js (hunk 4 above)
Comment (meloneminze) [reply client/.storybook/daylightTheme.js:19]: It is in the exapmle. Do you think I can delete it?

Comment (meloneminze) [reply client/.storybook/daylightTheme.js:19]: I will delete it. I had taken it over from the example.

File: client/.storybook/themeDecorator.js (hunk 5 above)
Comment (meloneminze) [reply client/.storybook/themeDecorator.js:8]: Ok.

**Pull request merged**